t/apicast-policy-headers.t

```diff
@@ -723,3 +723,62 @@ yay, api backend
 --- no_error_log
 [error]
 oauth failed with
+
+=== TEST 12: Liquid validate capture works
+Validate that capture function works correctly THREESCALE-1911
+--- backend
+  location /transactions/authrep.xml {
+    content_by_lua_block {
+      ngx.exit(200)
+    }
+  }
+--- configuration
+{
+  "services": [
+    {
+      "id": 42,
+      "backend_version":  1,
+      "backend_authentication_type": "service_token",
+      "backend_authentication_value": "token-value",
+      "proxy": {
+        "api_backend": "http://test:$TEST_NGINX_SERVER_PORT/",
+        "proxy_rules": [
+          { "pattern": "/", "http_method": "GET", "metric_system_name": "hits", "delta": 2 }
+        ],
+        "policy_chain": [
+          { "name": "apicast.policy.apicast" },
+          {
+            "name": "apicast.policy.headers",
+            "configuration":
+              {
+                "request":
+                  [
+                    {
+                      "op": "set",
+                      "header": "New-Header-1",
+                      "value": "{% capture foo%}bar{% endcapture%}{{foo}}",
```

Comment:
Can we limit the number of allocations using the liquid lib? Otherwise, this might not be safe.
Ref: https://github.com/3scale/APIcast/blob/f2c4168c5be272bc3e1a7f12c63736ffd94dad6d/gateway/src/apicast/template_string.lua#L70

Reply:
Does not look like that there is a way to limit it: 
https://github.com/chenxianyu2015/liquid-lua/blob/58e0a197445300538adab021d574f9375ebb6eba/lib/liquid.lua#L2610-L2620

And this one: 

https://github.com/chenxianyu2015/liquid-lua/blob/58e0a197445300538adab021d574f9375ebb6eba/lib/liquid.lua#L901